tools/enrollment_preprocessor.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import csv
+import sys
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from io import BytesIO
+from typing import TextIO
+
+from openpyxl import load_workbook
+from openpyxl.cell import Cell
+
+
+@dataclass
+class User:
+    title: str
+    last_name: str
+    first_name: str
+    email: str
+
+
+@dataclass
+class UserCells:
+    title: Cell | None
+    last_name: Cell
+    first_name: Cell
+    email: Cell
+
+    def value(self) -> User:
+        return User(
+            str(self.title.value) or "" if self.title else "",
+            str(self.last_name.value),
+            str(self.first_name.value),
+            str(self.email.value),
+        )
+
+
+def user_decision(field: str, existing: str, imported: str) -> str:
+    if existing == imported:
+        return existing
+    decision = input(f"Do you want to keep the existing user {field}? (y/n) ")
+    if decision and decision[0].lower() == "n":
```

Comment:
This makes "y" the default on any other input that "n", right? I'd say we want stricter input validation by repeatedly asking the user until the input is either empty (=default), "y", or "n". If someone entered "x", they likely made a mistake and possibly intended to type "y".

If we keep a "default for empty input", this should be shown on the prompt by using "(Y/n)"

@janno42 opinions? Do we want a "no input == yes" policy? I'd think we wouldn't want that.

(side note: I think we have similar "ask the user for yes/no" logic in multiple places. Could be deduplicated/shared)

Reply:
I wouldn't set a default for now and force the user to make a choice. If this turns out to be too annoying with real data, we can change it later.